### DISCUSSION TITLE
Базы для 5.4.8

### DISCUSSION THREAD

**Discussion by xBarselonx** [alexkulya/pandaria_5.4.8#95] [Q&A] [answered]
Добрый день!
А есть ссылка на базы данных?

**Comment by alexkulya** [comment, marked as answer]
Есть. Тут находятся всё необходимые базы данных. 
https://github.com/alexkulya/pandaria_5.4.8/tree/master/sql/base

**Reply by xBarselonx** [reply]
Ок. Спасибо.

**Reply by xBarselonx** [reply]
Я так понимаю в ручную базы заливать нужно? Архив волд базы кидать в папку с компилированными файлами? Пару комментариев можно получить?

**Reply by alexkulya** [reply]
Всё базы нужно заливать вручную. Из архива ворлд распаковываете базу ворлд и также заливаете.

**Reply by xBarselonx** [reply]
Понял. Спасибо!
